<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeStaffInfor</class>
 <widget class="QDialog" name="ChangeStaffInfor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>343</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>301</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>301</height>
      </size>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>161</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lnName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnPhone">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="inputMask">
          <string>9999999999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCompleted">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnPaid">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnUsername">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Admin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Staff</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbBoxGender">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAge">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnAddress">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>30</y>
        <width>71</width>
        <height>258</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Phone:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Completed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Total paid:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>User name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>User type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Gender:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lnName</tabstop>
  <tabstop>lnPhone</tabstop>
  <tabstop>spinBoxCompleted</tabstop>
  <tabstop>lnPaid</tabstop>
  <tabstop>lnUsername</tabstop>
  <tabstop>cbBoxType</tabstop>
  <tabstop>cbBoxGender</tabstop>
  <tabstop>spinBoxAge</tabstop>
  <tabstop>lnAddress</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
